packages/components/src/range-control/styles/range-control-styles.js

```diff
@@ -58,7 +58,7 @@ export const BeforeIconWrapper = styled.span`
 export const AfterIconWrapper = styled.span`
 	margin-top: ${ railHeight }px;
 
-	${ rtl( { marginLeft: 16 } ) }
+	${ rtl( { marginLeft: 6 } ) }
```

Comment:
This change seems fine since it matches the metrics of the BeforeIconWrapper as well.

Reply:
Cool, I just wasn't sure if the difference was on purpose or not.

Reply:
Thank yoU!